data/docs_markdown/Device_Block_Reward_Multiplier.md

```
@@ -0,0 +1,125 @@
+> Until **July 1, 2025** , staking requirements will continue to be calculated using the current earning multiplier.
+To qualify for block rewards, each device must meet a minimum staking threshold calculated as follows:
+  * **Base Stake** per processor: 200 $IO
+  * **Earning Multiplier** : Varies by device performance
+  * **Number of processors** : Total processors in the device
+
+
+###### [](https://docs.io.net/docs/proposed-device-block-reward-multiplier#minimum-stake--base-stake-%C3%97-max1-earning-multiplier-%C3%97-number-of-processors)
+Starting **July 1, 2025** , staking requirements may increase for some devices, as a new calculation logic will take effect. For example:
+  * A device with **8** H100 **GPUs** , each having an earning **multiplier of 10** :  
+Minimum Stake calculation: **200 × 10 × 8 = 16,000 $IO**
+  * A device with **4** RTX 4070 **GPUs** , each having an earning **multiplier of 0.25** :  
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 0; Actual: 2

(MD007, ul-indent)